nix/pkgs/libspdk/default.nix

```diff
@@ -55,8 +64,8 @@ stdenv.mkDerivation rec {
     "${enableFeature enableDebug "unit-tests"}"
     "--target-arch=nehalem"
     "--without-isal"
+    "--without-vhost"
```

Comment:
Presumably we don't use/need vhost at the moment, is there another reason this is being disabled?